db/migrate/20210520200350_create_ingress_payload.rb

```diff
@@ -0,0 +1,11 @@
+class CreateIngressPayload < ActiveRecord::Migration[5.2]
+  def change
+    create_table :ingress_payloads do |t|
+      t.string :task_id
+      t.string :request_id
+      t.jsonb :input
```

Comment:
@hsong-rh Should this is payload or input?

Reply:
changed it to `payload`